### DISCUSSION TITLE
Error 1406: Data too long for column

### DISCUSSION THREAD

**Discussion by DXorSX** [photoprism/photoprism#2151] [Q&A] [answered]
Hi,

I'm not sure if thats a bug or I'm just facing some hard limits. I got a lot of errors:
`time="2022-03-14T17:05:49Z" level=error msg="moments: Error 1406: Data too long for column 'album_slug' at row 1 (create folder)"`

My library has nearly 140.000 Images+Videos. Are there any known Limits? Or is it a bug?

**Comment by lastzero** [comment, marked as answer]
Seems a folder name is longer than expected. What is your longest folder name?

**Reply by DXorSX** [reply]
40 Characters is the longest folder name. The absolute path is much longer but from your question i guess thats not relevant.

**Reply by lastzero** [reply]
The path relative to your originals folder matters, not just the last folder name.

**Reply by DXorSX** [reply]
shortening the path by renaming some folders and re-indexing did the trick, thanks!

**Reply by lastzero** [reply]
see https://github.com/photoprism/photoprism/issues/2181
